package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "apostrophe",
-  "version": "3.27.0",
+  "version": "3.27.1",
```

```suggestion
  "version": "3.28.0",
```

if I'm not wrong, regarding the added features (webp, closing modals and checkboxes min and max fields)